### ISSUE TITLE
zablokování autoprint při více počítačích

### ISSUE THREAD

**Issue by otahirs** [fvacek/quickbox#513]
když se na druhém PC zaklikne autoprint, tak přestane fungovat.

---

situace:
1. počítač s QE
- připojena krabička a termotiskárna
- nastaven autoprint

2. počítač
- nic nepřipojeno
- nastaven autoprint

=> autoprint na PC 1 nefunguje.

**Comment by fvacek** [comment]
proc je na 2. pocitaci nastaven autoprint, kdyz tam neni tiskarna? Omylem?

**Comment by fvacek** [comment]
ja bych rekl, ze se QE snazi ty listecky vytisknout na nefunkcni tiskarne a na tu druhou se nedostane

**Comment by fvacek** [comment]
zkusil bych zaskrtnout "this printer only"

**Comment by otahirs** [comment]
> proc je na 2. pocitaci nastaven autoprint, kdyz tam neni tiskarna? Omylem?

ano, ale i tak bych nechtěl toto chování, kdo ví co mi ten vedle u počítače zaklikne

> zkusil bych zaskrtnout "this printer only"

vyzkoušené a stejné chování

**Comment by fvacek** [comment]
a jinak "this printer only" funguje?
